frontend/src/pages/generators/LetterGenerator.tsx

```
@@ -0,0 +1,471 @@
+              </div>
+
+              <div className="grid grid-cols-2 md:grid-cols-3 gap-4">
+                {letterTemplates.map((t) => (
+                  <div
+                    key={t.id}
+                    onClick={() => setSelectedTemplateId(t.id)}
+                    className={`relative group cursor-pointer aspect-[3/4] rounded-3xl border-2 transition-all overflow-hidden ${selectedTemplateId === t.id
+                      ? "border-primary bg-primary/10 shadow-[0_0_30px_rgba(var(--primary),0.2)]"
+                      : "border-border bg-card/40 hover:border-primary/50"
+                      }`}
+                  >
```

Biome (2.3.13)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)